### DISCUSSION TITLE
Reducing Costs for Updating Provider Attributes on the Akash Network

### DISCUSSION THREAD

**Discussion by devalpatel67** [akash-network/community#486] [Governance Proposals]
**Executive Summary**
This proposal addresses the financial burden placed on providers within the Akash Network when updating their provider attributes, such as domain names and other associated values. Currently, the cost for performing such updates is approximately 3.5 AKT, which, with the current AKT price hovering around USD 3.65, represents a significant expense. This proposal advocates for a reduction in the cost associated with these transactions to support providers in maintaining accurate and up-to-date service offerings, thereby enhancing the overall efficiency and competitiveness of the Akash Network.

**Proposal**
We propose revising the transaction cost structure to update the Akash Network's provider attributes. Specifically, we suggest that the cost of transactions related to modifying provider attributes, including but not limited to domain name changes and the addition of new attributes, be significantly reduced.

**Justification**
Reducing the transaction costs for updating provider attributes is justified on several grounds:

1. **Encourages Accurate and Up-to-date Information**: Lowering the cost will encourage providers to keep their service offerings current without the deterrent of a high fee. This ensures that consumers on the Akash Network have access to the latest and most accurate information, facilitating better decision-making.

2. **Supports Provider Flexibility and Innovation**: A lower transaction cost removes a barrier for providers wishing to innovate or expand their services. This flexibility can lead to a more dynamic and competitive marketplace.

3. **Enhances Network Competitiveness**: By reducing operational frictions, the Akash Network can become more attractive than centralized alternatives, attracting more providers and users to the platform.

4. **Addresses Financial Burden**: With the current AKT price, the burden of updating provider attributes is disproportionate. Reducing this cost acknowledges providers' economic realities and supports their continued participation in the network.

**Implementation**
To implement this proposal, we recommend the following steps:
1. **Technical Assessment**: Conduct a thorough assessment to determine the optimal new cost for updating provider attributes, ensuring it covers the network's operational needs without unduly burdening providers.

2. **Community Consultation**: Engage with the Akash community, including providers and users, to gather feedback on the proposed cost structure and any potential impacts on the network.

3. **Rollout Adjustment**: If approved, introduce the new cost structure, review the outcomes and make any necessary adjustments based on feedback and observed effects on the Akash Network ecosystem.

**Conclusion**
This proposal aims to strike a balance between the operational needs of the Akash Network and the economic realities faced by providers. By reducing the cost of updating provider attributes, we can foster a more dynamic, competitive, and user-friendly ecosystem. We'd like to invite the community to engage in a constructive dialogue on this proposal, with the ultimate goal of enhancing the value and appeal of the Akash Network for all participants.

**Comment by rodri-r** [comment]
Hello, I agree with this,although I do not believe it currently costs 3.5 AKT to update provider attributes. Last time I updated my provider's attributes, it was around 1 AKT. (This was last year.)

You probably updated it 3 times without knowing (it happened to me too)

So workaround, if current cost is not updated: 

1. there should be a warning or note somewhere before updating provider's info
2. have your provider's info ready before pushing the updated info.

But otherwise, yes maybe it should cost a little less, now that AKT price is back up.

**Reply by ghost** [reply]
Is there a way to peg this cost to a specific USD amount? 
that way the price wont change as AKT goes up or down depending on the market

**Reply by vatsal2809** [reply]
Yes one of the provider updated the attribute today and it cost him 3.5AKT

![image](https://github.com/akash-network/community/assets/99423464/fb3a1653-bb9e-4e04-b7e0-fbda46fefb59)

**Reply by rodri-r** [reply]
This is already being talked about and there will most likely be some changes soon.

**Reply by devalpatel67** [reply]
It is being taken care of at - https://www.mintscan.io/akash/proposals/249 this proposal.
